doc/testing_guide.md

```diff
@@ -78,7 +78,7 @@ yarn start
 ### Local parachain (collator)
 Download the `polka-storage` parachain repository:
 ```bash
-git clone https://github.com/eigerco/polka-storage.git
+git clone https://github.com/eigerco/polkadot-native-storage polka-storage
```

Comment:
and here.

Reply:
Let's leave it as the first message says.

Reply:
👍